.prettierignore

```diff
@@ -2,3 +2,4 @@
 coverage/
 node_modules/
 lib/
+README.md
```

Comment:
I might make sense to use Prettier to format the entire project, but I didn't want to bloat this diff.

It looks like my editor did remove some trailing spaces (which I'm happy to revert if you want).